core/tests/protractor/learnerDashboard.js

```diff
@@ -19,7 +19,8 @@
 var AdminPage = require('../protractor_utils/AdminPage.js');
 var CreatorDashboardPage =
   require('../protractor_utils/CreatorDashboardPage.js');
-var collectionEditor = require('../protractor_utils/collectionEditor.js');
+var CollectionEditorPage = 
```

Comment:
@paroothisumit, your PR is failing Travis lint test please delete the trailing spaces from this line.